identity-enabler/deviceId-mobile-app/src/models/types/IInvalidCredentialPageState.ts

```diff
@@ -0,0 +1,5 @@
+export interface IInvalidCredentialPageState extends Record<string | number, unknown> {
```

Comment:
why does it need to extend Record?

Reply:
Because the `navigate` function from `svelte-routing` defines the `state` parameter as 
```js
{
      [k in string | number]: unknown;
}
```

Reply:
a comment would be necessary